CondCore/Utilities/test/test_getPayloadData.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+check_for_success() {
+    "${@}" && echo -e "\n ---> Passed test of '${@}'\n\n" || exit 1
+}
+
+check_for_failure() {
+    "${@}" && exit 1 || echo -e "\n ---> Passed test of '${@}'\n\n"
+}
+
+check_for_full(){
+    count=`echo ${@} | python3 -c 'import json,sys;print(len(json.load(sys.stdin)))'`
```

Comment:
@mmusich , I think you might want to get the length of `cond::BasicPayload` e.g.
```
python3 -c 'import json,sys;print(len(json.load(sys.stdin)["cond::BasicPayload"]))'
```
otherwise length is already `1`